releases/release-4.0.8.md

```diff
@@ -0,0 +1,161 @@
+---
+title: TiDB 4.0.8 Release Notes
+---
+
+# TiDB 4.0.8 Release Notes
+
+Release date: October 29, 2020
+
+TiDB version: 4.0.8
+
+## New Features
+
++ TiDB
+
+    - Support new aggregate function APPROX_PERCENTILE [#20197](https://github.com/pingcap/tidb/pull/20197)
+
++ TiFlash
+
+    - Support CAST functions push down
+
++ Tools
+
+    + TiCDC
+
+        - Supports snapshot level consistency replication in time [#932](https://github.com/pingcap/ticdc/pull/932)
+
+## Improvements
+
++ TiDB
+
+    - Prioritize low selectivity indexes in the greedy search procedure of `Selectivity()` [#20154](https://github.com/pingcap/tidb/pull/20154)
+    - Record more RPC runtime information in cop runtime stats [#19264](https://github.com/pingcap/tidb/pull/19264)
+
++ TiKV
+
+    - Add the Fast-Tune panel page as performance assisted diagnosis [#8804](https://github.com/tikv/tikv/pull/8804)
+    - Add `security.redact-info-log` config, which redacts user data from logs [#8746](https://github.com/tikv/tikv/pull/8746)
+    - Error_code: re-format metafile [#8877](https://github.com/tikv/tikv/pull/8877)
+    - Enable dynamically change config pessimistic-txn.pipelined [#8853](https://github.com/tikv/tikv/pull/8853)
+    - Enable profiling features by default [#8801](https://github.com/tikv/tikv/pull/8801)
+
++ PD
+
+    - Generate the metafile of errors [#3090](https://github.com/pingcap/pd/pull/3090)
+    - Add additional info for operator [#3009](https://github.com/pingcap/pd/pull/3009)
+
++ TiFlash
+
+    - Add metrics about applying Raft logs
+    - Add metrics about memory usage for cop task
+    - Make the min-max index more accurate if there are deleted data
+    - Improved query performance under small volume data
+    - Add errors.toml to support standard error code
+
++ Tools
+
+    + TiCDC
+
+        - Print statistics in MySQL sink [#1023](https://github.com/pingcap/ticdc/pull/1023)
+
+    + Backup and Restore (BR)
+
+        - Speed up restore by pipeline split and restore [#428](https://github.com/pingcap/br/pull/428)
```

Comment:
@overvenus This note seems inconsistent with the #428 PR. Could you help confirm?